tests/test_adapter_heads.py

```diff
@@ -229,6 +229,8 @@ def test_batch_split_head(self):
         in_data = self.get_input_samples((3, 128), config=model.config)
 
         out = model(in_data)
+        out = out.head_outputs
```

Comment:
Maybe there is a possible to let `MultiHeadOutput` behave like a list so we don't have to access head_outputs explicitly?